src/databricks/labs/lsql/dashboards.py

```diff
@@ -919,3 +919,17 @@ def _replace_names(self, node: T, better_names: dict[str, str]) -> T:
             if node.spec is not None:
                 node.name = node.spec.as_dict().get("widgetType", node.name)
         return node
+
+    def get_dashboard_url(self, dashboard: SDKDashboard) -> str:
```

Comment:
```suggestion
    def get_url(self, dashboard_id: str) -> str:
```
don't leak internal types :)